apps/toolkit-app/src/pages/api/rpc/[[...blitz]].ts

```diff
@@ -1,4 +1,20 @@
 import { rpcHandler } from "@blitzjs/rpc"
 import { api } from "src/blitz-server"
 
-export default api(rpcHandler({ onError: console.log }))
+class ErrorWithStatusCode extends Error {
```

Comment:
Hey @dbrxnds , This looks good, I feel it would be better to export this error maybe something like `BlitzErrorWithStatusCode` in the main blitz package.
This would make it easier to maintain and reuse as the code is now hardcoding `status Code` attribute to be present in the custom error for it to work.

Reply:
This is only inside the tookit-app for testing now. Would you still prefer it to be exported from blitz package? Or should I just remove it from this code so people can make up their own logic for it?